.circleci/lighthouse.yml

```
@@ -1,19 +1,25 @@
+      - run: sudo pnpm install -g @lhci/cli@0.14.x
+      - run: lhci autorun
 
 workflows:
-  lhci-test:
+  run-lhci:
     jobs:
-      - lhci-test
\ No newline at end of file
+      - build:
+          context:
+            - Main Context
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)